cfgov/templates/wagtailadmin/css/simple-chart-admin.css

```diff
@@ -0,0 +1,33 @@
+  @media (min-width: 970px) {
+  .struct-block.simple-chart-block {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: -1.5em;
+  }
+
+  .struct-block.simple-chart-block > .field {
+    min-width: calc(50% - 1.5em);
+    margin-right: 1.5em;
+  }
+
+  .struct-block.simple-chart-block > .field + .field {
+    padding-top: 0;
+    padding-bottom: 0.5em;
+  }
+  
+  .struct-block.simple-chart-block > .field[data-contentpath="data_series"] {
+    min-width: calc(100% - 1.5em);
+  }
+
+  .struct-block.simple-chart-block > .field[data-contentpath="filters"] {
+    min-width: calc(100% - 1.5em);
+  }
+
+  .struct-block.simple-chart-block > .field[data-contentpath="style_overrides"] {
+    min-width: calc(100% - 1.5em);
+  }
+
+  .struct-block.simple-chart-block > .field[data-contentpath="download_file"] {
+    margin-left: 0;
+  }
+ }
```

Comment:
The formatting is off in this file and now that we have stylelint (!!) we can autoformat it. In package.json try changing [L67](https://github.com/cfpb/consumerfinance.gov/blob/0b949a2b261de160b51b57b820c39a7bf65e1810/package.json#L67) to:

```js
"lint": "eslint --ignore-pattern node_modules './{cfgov/unprocessed,config,esbuild,scripts,test}/**/*.js' --fix && stylelint './cfgov/{unprocessed,templates/wagtailadmin/css}/**/*.{css,less}' --fix",
```

and then run `yarn lint` to fix all our custom wagtail admin CSS files along with our normal Less files.